src/openlcb/SimpleStack.cxx

```diff
@@ -193,6 +194,7 @@ void SimpleTrainCanStack::start_node()
 void SimpleStackBase::start_after_delay()
 {
     start_iface(false);
+    node()->initialize();
```

Comment:
i would expect to iterate the iface()->first_local_node() / next_local_node() list and call initialize on all of them. This would ensure that all virtual nodes get correctly initialized as well.

Reply:
I have made this update as suggested.